### DISCUSSION TITLE
Unable to merge code from branch to main branch

### DISCUSSION THREAD

**Discussion by DOH-REP1303** [community/community#47274] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am trying to merge code from my coding branch to my main branch. When I try, I get the error message below.
Remote: warning: File SSIS/ExportFiles/PatientInfo.csv is 85.82 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB
Remote: Error: Trace: 543219d8ffaacea6ec01cdecee925b75eb62e5881b7587a0a6ab3d3eae240f99
Remote: Error: See http://git.io/iEPt8g for more information.
Remote: Error: File SSIS/ExportFiles/CaseIX.csv is 275.25 MB; this exceeds GitHub's file size limit of 100.00 MB
Remote: Error: GH001: Large files detected. You may want to try Git Large File Storage - https://git-lfs.github.com/.
To https://github.com/DOH-HTS-ADS/PHIMS-STD.git
Error: Version_831 -> Version_831 (pre-receive hook declined)
failed to push some refs to 'https://github.com/DOH-HTS-ADS/PHIMS-STD.git'

I tried to remove the file from my local commit. This was unsuccessful because the file does not exist on my local hard drive. I cannot figure out where SSIS/ExportFiles/CaseIX.csv is located because the full path is not given. All the potential solutions require the full file path name. How do I solve this issue? I even recreated the directory on my local hard drive so that I have the file present. That does not work either. What really does not make sense is I am not checking in that portion of the code. Thank you for any help and suggestions. Rosalind

**Comment by airtower-luna** [comment]
Most likely the file still exists in the repository history (some older commit). Deleting it and committing isn't enough if the file was previously committed. The process to remove the file entirely from the repository history is [the same as for sensitive files](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository). If you want to learn about modifying history in general, see [Git Tools - Rewriting History](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History).

**Reply by DOH-REP1303** [reply]
I still could not delete the file from the repository history because the command still wanted the path of the actual file. What I ended up doing was backing up my code to another directory, Deleting the code from the original directory, cloning the code back into the directory checked out, deleting the code and pasting in the code from the backed up directory. I was then able to push the code successfully. Thank you @airtower-luna for your help.
